app/components/widgets/forms/file-upload.js

```diff
@@ -0,0 +1,101 @@
+import Ember from 'ember';
+import { humanReadableBytes, isFileValid } from 'open-event-frontend/utils/file';
+import { v4 } from 'ember-uuid';
+
+const { Component, computed } = Ember;
+
+export default Component.extend({
+
+  selectedFile  : null,
+  allowDragDrop : true,
+
+  inputIdGenerated: computed('inputId', function() {
+    return this.get('inputId') ? this.get('inputId') : v4();
+  }),
+
+  maxSize: computed('maxSizeInKb', function() {
+    return humanReadableBytes(this.get('maxSizeInKb'));
+  }),
+
+  uploadFile() {
+    this.set('needsConfirmation', false);
+    this.set('uploadingFile', true);
+    this.get('loader')
+      .uploadFile('/upload/files', this.$(`#${this.get('inputIdGenerated')}`))
+      .then(file => {
```

Comment:
@geekyd Please display the file name once the file is uploaded, so that the user knows if the upload is complete.